### PR TITLE
COMMERCE-6975 Order type field 1/3 (1 of 2)

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
use **generated buildUpgradeTable** pattern

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -165,6 +165,9 @@ components:
                     allOf:
                         - $ref: "#/components/schemas/Status"
                     readOnly: true
+                orderTypeId:
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/rest-openapi.yaml:168]: please move rest changes in a different PR

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v7_1_0/CommerceOrderUpgradeProcess.java [hunk 2]
```diff
@@ -15,18 +15,19 @@
...
+ * @author Alessio Antonio Rendina
  */
 public class CommerceOrderUpgradeProcess
 	extends BaseCommerceServiceUpgradeProcess {
 
 	@Override
 	protected void doUpgrade() throws Exception {
 		addColumn(
-			CommerceOrderModelImpl.class, CommerceOrderModelImpl.TABLE_NAME,
+			CommerceOrderTable.class, CommerceOrderTable.TABLE_NAME,
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v7_1_0/CommerceOrderUpgradeProcess.java:30]: please fixup commit

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v7_1_0/util/CommerceOrderTable.java [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+/**
+ * @author	  Brian Wing Shun Chan
+ * @generated
+ */
+public class CommerceOrderTable {
+
+	public static final String TABLE_NAME = "CommerceOrder";
+
+	public static final Object[][] TABLE_COLUMNS = {
+		{"uuid_", Types.VARCHAR}, {"externalReferenceCode", Types.VARCHAR},
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v7_1_0/util/CommerceOrderTable.java:31]: does it make sense to move this to the autogenerated code commit?

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:reevaluate:1267401_143

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**